src/ToolServiceProvider.php

```diff
@@ -64,11 +61,32 @@ protected function routes()
 
     /**
      * Register any application services.
-     *
-     * @return void
      */
-    public function register()
+    public function register(): void
     {
-        $this->app->bind(PublishManager::class, PublishManager::class);
+        $this->app
+            ->when(GitHubApi::class)
+            ->needs('$owner')
+            ->giveConfig("publish.owner");
```

Comment:
Wow, dit is een mooie api. Deze kende ik nog niet.

Reply:
Dit kan tegenwoordig ook met een attribuut! 
Zie https://laravel.com/docs/11.x/container#contextual-attributes

Reply:
Alleen vrees ik dat lagere Laravel versies dit niet vreten.